packages/components/pie-webc-testing/src/playwright/fixtures.ts

```diff
@@ -13,8 +13,8 @@ interface ExtendedTestContext {
 export const test = baseTest.extend<ExtendedTestContext>({
     makeAxeBuilder: [async ({ page }: { page: Page }, use: (builder: () => AxeBuilder) => Promise<void>) => {
         const makeAxeBuilder = () => new AxeBuilder({ page })
-        .withTags(['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria'])
-        .disableRules(['color-contrast-enhanced']);
+          .withTags(['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria'])
```

Comment:
What was missing to make the indentation work?

Reply:
i did it manually :D